lib/Net/SAML2/Protocol/AuthnRequest.pm

```diff
@@ -147,6 +151,20 @@ has 'RequestedAuthnContext_Comparison' => (
     default => 'exact'
 );
 
+has 'force_authn' => (
+    isa     => 'Str',
+    is      => 'rw',
```

Comment:
I would also make these ro instead of rw

Reply:
Will fix